app/src/pages/inside/testCaseLibraryPage/emptyState/folder/folderEmptyState.tsx

```diff
@@ -15,38 +15,59 @@
  */
 
 import Parser from 'html-react-parser';
+import { FC, SVGProps } from 'react';
 import { useIntl } from 'react-intl';
 
 import { EmptyStatePage } from 'pages/inside/common/emptyStatePage';
 import ImportIcon from 'common/img/import-thin-inline.svg';
 import PlusIconInline from 'common/img/plus-button-inline.svg';
+import { useUserPermissions } from 'hooks/useUserPermissions';
 
 import { messages } from '../messages';
 import { commonMessages } from '../../commonMessages';
 
+interface Button {
```

Comment:
Define this type once and use in all other similar places. For example I see here you defined it in [another way](https://github.com/reportportal/service-ui/pull/4578/files). There are other places too